react-site/src/visualizations/LineGraph/LineGraph.js

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import * as d3 from "d3";
+
+import { useD3 } from "../../hooks/useD3";
+
+import Card from "../../components/Card";
+
+import "./LineGraph.css";
+
+export const LineGraph = ({
```

Comment:
In the line graphs, it's confusing how 2021 is underneath the graph as well as the months. Can we replace 2021 with january and put 2021 underneath january?

Reply:
Technically, no 😅

D3 manages scale automatically which works in most cases, and since we have a tooltip that shows actual values I think it's fine here